packages/std/src/binary.rs

```diff
@@ -165,6 +165,34 @@ impl PartialEq<Binary> for &[u8] {
     }
 }
 
+/// Implement `Binary == &[u8; LENGTH]`
```

Comment:
Won't `forward_ref_binop!` handle this implementation?

Reply:
I just had another look. forward_ref_binop requirey types to be Copyable. So we can't use it here. Writing our own macro will also not be easy since we need an impl with generic type for `impl<const LENGTH: usize> PartialEq<[u8; LENGTH]> for Binary`.